client/src/components/Q&A/QA.jsx

```diff
@@ -2,17 +2,19 @@ import React, { useState } from 'react';
 import SearchBar from './SearchBar';
 import QuestionContainer from './QuestionContainer';
 import AddQModal from './AddQModal';
+import AddAnswerModal from './AddAnswerModal';
 
-export default function QA({ show, setShow }) {
+export default function QA() {
   const [searchTerm, setSearchTerm] = useState('');
   return (
     <div
       className="main-container"
     >
       <h2>Questions & Answers</h2>
-      <AddQModal show={show} setShow={setShow} />
+      <AddQModal />
       <SearchBar setSearchTerm={setSearchTerm} />
-      <QuestionContainer searchTerm={searchTerm} setShow={setShow} />
+      <AddAnswerModal />
+      <QuestionContainer searchTerm={searchTerm} />
```

Comment:
since we figured out the bug with the add answer modal, maybe move it back up to be next to the add q modal, just for better readability